tests/unit/backend/corpora/common/entities/test_dataset.py

```diff
@@ -93,16 +94,51 @@ def test__update__ok(self):
             s3_uri="some_uri",
         )
         deployment_directory_params = dict(url="test_url")
+        processing_status = dict(
```

Comment:
to avoid duplication can you make a BogusProcessingStatus Class [here]( https://github.com/chanzuckerberg/corpora-data-portal/blob/ceb8955a9bcecfca6d47d3181741c7541eec4d31/tests/unit/backend/utils.py#L6) and use it here.